hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -56,6 +56,7 @@
   /** Set this key to {@code true} to enable metrics collection of client requests. */
   public static final String CLIENT_SIDE_METRICS_ENABLED_KEY = "hbase.client.metrics.enable";
 
+  private static final String CNT_BASE = "rpcCount_";
```

Comment:
Is this an unrelated change?

Reply:
I see. Would be weird to do different. Thought it was its own commit in master history. np.